core/src/main/resources/reference.conf

```diff
@@ -156,7 +156,25 @@ datastax-java-driver {
     # - com.datastax.dse.driver.internal.core.loadbalancing.
     # - com.yugabyte.oss.driver.internal.core.loadbalancing.
     #
-    # The driver provides a single implementation out of the box: DefaultLoadBalancingPolicy.
+    # The driver provides three implementations out of the box:
+    #
+    # - `DefaultLoadBalancingPolicy`: should almost always be used; it requires a local datacenter
+    #   to be specified either programmatically when creating the session, or via the configuration
+    #   option: datastax-java-driver.basic.load-balancing-policy.local-datacenter. It can also
+    #   use a highly efficient slow replica avoidance mechanism, which is by default enabled – see
+    #   the option: datastax-java-driver.basic.load-balancing-policy.slow-replica-avoidance.
+    # - `DcInferringLoadBalancingPolicy`: similar to `DefaultLoadBalancingPolicy`, but does not
+    #   require a local datacenter to be defined, in which case it will attempt to infer the local
+    #   datacenter from the provided contact points, if possible; if that fails, it will throw an
+    #   error during session initialization. This policy is intended mostly for ETL tools and
+    #   should not be used by normal applications.
+    # - `BasicLoadBalancingPolicy`: similar to `DefaultLoadBalancingPolicy`, but does not have
+    #   the slow replica avoidance mechanism. More importantly, it is the only policy capable of
+    #   operating without local datacenter defined, in which case it will consider nodes in the
+    #   cluster in a datacenter-agnostic way. Beware that this could cause spikes in
+    #   cross-datacenter traffic! This policy is provided mostly as a starting point for users
+    #   wishing to implement their own load balancing policy; it should not be used as is in normal
+    #   applications.
```

Comment:
Add a comment explaining `PartitionAwarePolicy`similar to the other LoadBalancing policies in `reference.conf`.

Reply:
Will do.